src/SophiApp/Localizations/IT.xaml

```diff
@@ -31,31 +31,31 @@
     <system:String x:Key="Localization.Security">Sicurezza</system:String>
     <system:String x:Key="Localization.ContextMenu">Menu contestuale</system:String>
     <system:String x:Key="Localization.Settings">Pulizia di Windows</system:String>
     <system:String x:Key="Localization.ViewApplyingException.InApplying">Mentre si applicavano le modifiche</system:String>
-    <system:String x:Key="Localization.ViewApplyingException.HasException">c'è stato un'errore.</system:String>
+    <system:String x:Key="Localization.ViewApplyingException.HasException">si è verificato un'errore.</system:String>
     <system:String x:Key="Localization.CleanupTask.NotificationTask.Title">Informazioni importanti</system:String>
-    <system:String x:Key="Localization.CleanupTask.NotificationTask.EventTitle">Esguire una pulizia dei file temporanei e degli aggiornamenti di Windows già installati?</system:String>
-    <system:String x:Key="Localization.CleanupTask.Description">Cancellazzione dei file temporanei e degli aggiornamenti di Windows già installati con l'app Pulizia disco.</system:String>
-    <system:String x:Key="Localization.CleanupTask.NotificationTask.Description">Notifiche Pop-up per ricordarti di cancellare i file di Windows inutilizzati e gli aggiornamenti già installati.</system:String>
-    <system:String x:Key="Localization.CleanupTask.NotificationTask.Run">esegui</system:String>
-    <system:String x:Key="Localization.SoftwareDistributionTask.Event">La cache di Windows update è stata eliminata.</system:String>
-    <system:String x:Key="Localization.SoftwareDistributionTask.Description">pulendo la cartella %SystemRoot%\SoftwareDistribution\Download.</system:String>
-    <system:String x:Key="Localization.ClearTempTask.Event">La cartella dei file temporanei è stata pulita.</system:String>
-    <system:String x:Key="Localization.ClearTempTask.Description">è in corso la pulizia della cartella dei file temporanei.</system:String>
+    <system:String x:Key="Localization.CleanupTask.NotificationTask.EventTitle">Vuoi eseguire una pulizia dei file temporanei e degli aggiornamenti di Windows già installati?</system:String>
+    <system:String x:Key="Localization.CleanupTask.Description">Cancellazione dei file temporanei e degli aggiornamenti di Windows già installati con l'app Pulizia disco.</system:String>
+    <system:String x:Key="Localization.CleanupTask.NotificationTask.Description">Notifiche popup per ricordarti di cancellare i file di Windows non usati e gli aggiornamenti già installati.</system:String>
+    <system:String x:Key="Localization.CleanupTask.NotificationTask.Run">Esegui</system:String>
+    <system:String x:Key="Localization.SoftwareDistributionTask.Event">La cache di Windows Update è stata eliminata.</system:String>
+    <system:String x:Key="Localization.SoftwareDistributionTask.Description">Pulizia della cartella %SystemRoot%\SoftwareDistribution\Download.</system:String>
```

Comment:
add in corso at the end

Reply:
I beleive that "..." ath the end is more modern then "in corso".

Reply:
Does not specify action is running

Reply:
For me "in corso" and "..." are both progressive action meaning.
In the past the translator prefer to sue "in corso" and now "..."
Ok for "in corso".

Reply:
I'm the translator... BTW, I still prefer in corso(for example take inspiration from windows translation).